pkg/process/process_manager.go

```diff
@@ -104,6 +136,61 @@ func (pm *Manager) startMonitoring() {
 	}
 }
 
+func (pm *Manager) checkMountPointStatusForEngine() {
```

Comment:
Suggest adding unit tests, even though it's a private method. This way we can better understand how mount points will be parsed here.

Reply:
I have move the mount point parsing and process name parsing out of the function and make it simpler to understand. With that, I think we don't need unit tests for it now.
Please take a look, thanks!